plugins/labhub.py

```diff
@@ -387,3 +388,63 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'pr\s+list\s+https://(github|gitlab)\.com/([^/]+)/([^/]+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='pr list <complete-repo-URL>',
+               flags=re.IGNORECASE)
+    def pr_list(self, msg, match):
+        """List PRs ready to be merged."""  # Ignore QuotesBear
+        host, org, repo_name = match.groups()
+
+        try:
+            mrs = self.REPOS[repo_name].merge_requests()
+        except KeyError:
+            return 'Repository doesn\'t exist.'
+        else:
+            checks = []
+
+            def check_mr(function):
+                checks.append(function)
+                return function
+
+            @check_mr
+            def check_labels(mr):
+                labels = mr.closes_issue.labels
```

Comment:
Should be checking MR labels `mr.labels` instead

Reply:
Fixed it!

Reply:
There's no mr.labels func in IGitt :(

Reply:
Maybe I should add an issue in IGitt repo?

Reply:
Are you sure, it should be! `MergeRequest` classes inherit from `Issue` classes, and hence, `labels` property should be available in `MergeRequest`s as well

Reply:
For some reason, it doesn't work with mr.labels, can't find out why

Reply:
Seems like I found the bug, will push my changes in a minute :)

Reply:
Yup, got everything fixed, it works with mr.labels now